src/app/features/config/default-global-config.const.ts

```diff
@@ -99,6 +99,7 @@ export const DEFAULT_GLOBAL_CONFIG: GlobalConfigState = {
     expandSubTasks: null,
     collapseSubTasks: null,
     togglePlay: 'y',
+    taskEditTags: 'z',
```

Comment:
```suggestion
    taskEditTags: 'g',
```
I prefer 'g' because of "taG" and because z is very far away for the left hand on a QWERTZ keyboard.